src/bindings/python/CMakeLists.txt

```diff
@@ -31,28 +44,74 @@ set_target_properties(valhalla PROPERTIES POSITION_INDEPENDENT_CODE ON)
 if(NOT VALHALLA_PYTHON_PACKAGE)
   set(VALHALLA_PYTHON_PACKAGE "pyvalhalla-weekly")
 endif()
+
 target_compile_definitions(_valhalla PRIVATE -DVALHALLA_PYTHON_PACKAGE=${VALHALLA_PYTHON_PACKAGE})
 
-# copy valhalla python sources from src to build
 set(py_srcs 
   valhalla/__init__.py
+  valhalla/__version__.py
```

Comment:
this will actually only be available for a scikit-build-core build, not a CMake one. further down in the "else" we write one to disk for the pure CMake build